### DISCUSSION TITLE
My marquee component and fabric.util If requestanimframe is used together, an exception will appear, causing the style of multiple check boxes not to be displayed

### DISCUSSION THREAD

**Discussion by w934423231** [fabricjs/fabric.js#8127] [Q&A] [answered]
My marquee component and fabric.util If requestanimframe is used together, an exception will appear, causing the style of multiple check boxes not to be displayed

**Comment by ShaMan123** [comment, marked as answer]
no one can understand this

**Reply by w934423231** [reply]
I will try to illustrate this problem, but in the form of pictures, I cannot accurately show the abnormality

**Reply by w934423231** [reply]
![image](https://user-images.githubusercontent.com/38958128/183320428-7d96dda3-79d6-48e2-84ac-55518e57f01b.png)
I execute the animation in the global environment. I can ensure that the box selection is faulty after the animation is added

**Reply by w934423231** [reply]
I found out where my problem was, but I didn't find a better solution

**Reply by w934423231** [reply]
![image](https://user-images.githubusercontent.com/38958128/183326900-6094310a-657a-42dc-b1c1-417d79c40f07.png)
Because renderall is called frequently, the canvas is refreshed repeatedly, and the canvas selection box will be lost during refreshing
